{{cookiecutter.repo_name}}/gunicorn.conf.py

```diff
@@ -0,0 +1,8 @@
+import multiprocessing
+
+accesslog = "-"
+bind = "0.0.0.0:8080"
+preload_app = True
+worker_class = "uvicorn.workers.UvicornWorker"
+workers = multiprocessing.cpu_count() * 2 + 1
```

Comment:
I would just add this as a comment above this line: https://docs.gunicorn.org/en/stable/design.html#how-many-workers